src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -73,11 +78,24 @@ const ProfileSelector = ({ input, showSelector }) => {
   };
 
   return (
-    <FormGroup
-      isRequired={true}
-      label={'Profile to use for this image'}
-      data-testid="profiles-form-group"
-    >
+    <FormGroup isRequired={true} data-testid="profiles-form-group">
+      {'OpenSCAP profile'}
+      <Popover
```

Comment:
Let's actually just delete this popover for now, since we don't have good copy for it.

Reply:
I asked Kelsea about the content of OpenScap Profile when we go over it and she gave me this content    'To run a manual compliance scan in OpenSCAP, download this image.' i will change it